gluten-celeborn/src/main/scala/org/apache/spark/shuffle/CelebornHashBasedColumnarShuffleWriter.scala

```diff
@@ -14,16 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
the style change iss not related to this pr. I just run `mvn spotless:apply -Pbackends-velox -Prss -DskipTests -Pspark-3.3`